documentation/modules/rn-2.3.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+[id="rn-23_{context}"]
+= {project-full} 2.3
+
+You can migrate virtual machines (VMs) from VMware vSphere or {rhv-full} to {virt} with {the-lc} {project-first}.
+
+The release notes describe technical changes, new features and enhancements, and known issues.
+
+[id="technical-changes-22_{context}"]
+== Technical changes
+
+This release has the following technical changes:
+
+.Setting the VddkInitImage path is part of the procedure of adding VMware provider.
+
+In the web console, you enter the VddkInitImage path when adding a VMware provider. Alternatively, from the CLI, you add the VddkInitImage path to the `Provider` CR for VMware migrations.
+
+.The `StorageProfile` resource needs to be updated for a non-provisioner storage class
+
+You must update the `StorageProfile` resource with `accessModes` and `volumeMode` for non-provisioner storage classes such as NFS. The documentation includes a link to the relevant procedure.
+
+[id="new-features-and-enhancements-22_{context}"]
+== New features and enhancements
+
+This release has the following features and improvements:
+
```

Comment:
Needs to be formatted line 34

Reply:
@RichardHoch  What do you mean?

Reply:
Line 30 Should be formatted like line 34. Line 30 does not stand out like line 34. How does it look when you use bccutil ?

Reply:
It is formatted the same.